lib/md055.js

```diff
@@ -0,0 +1,26 @@
+// @ts-check
+
+"use strict";
+
+const { addErrorContext } = require("../helpers");
+const { filterByTypes } = require("../helpers/micromark.cjs");
+
+module.exports = {
+    "names": [ "MD055", "table-missing-border" ],
+    "description": "Table is missing leading or trailing pipe character",
+    "tags": [ "table" ],
+    "function": function MD055(params, onError) {
+        const tables = filterByTypes(params.parsers.micromark.tokens, ["table"]);
+        for (const table of tables) {
+            const rows = filterByTypes(table.children.flatMap((t) => t.children), ["tableRow", "tableDelimiterRow"]);
+            for (const row of rows) {
```

Comment:
Nitpick: The first line inside the for block could be: `const { startLine, text } = row;` and then shorten the 6 following uses.